### PR TITLE
added credits scene to menu

### PR DESCRIPTION
**Pull request by lsizemore8**
Added the credits scene as a menu option. Also fixed the issue where the "Game Over" text sometimes was white or grey when you died, because it didn't seem worthwhile to make a separate PR for 1 line of code.

Note: I don't know why the savedata keeps on being modified to github when no new levels are being unlocked, so I gave up and just committed it.

### CODE REVIEW

**Comment by BryanGraeser** [comment]
Waiting for #153 save.json changes to be made and merged into master so this branch can be brought up-to-date without save.json changes

File: core/src/com/fallenflame/game/GameEngine.java [hunk 1]
```diff
@@ -509,6 +509,7 @@ public void draw(float delta) {
         } else if (isFailed) {
             canvas.beginWithoutCamera(); // DO NOT SCALE
             canvas.draw(border, canvas.getWidth()/2-(border.getRegionWidth()/2), canvas.getHeight()/2-(border.getRegionHeight()/2));
+            displayFont.setColor(Color.CYAN);
```
Comment (BryanGraeser) [inline core/src/com/fallenflame/game/GameEngine.java:512]: Why?

**Review by BryanGraeser — COMMENTED** [review]
Overall tested and it looks good. See my comment above and 1 question below.

File: core/src/com/fallenflame/game/GameEngine.java (hunk 1 above)
Comment (lsizemore8) [reply core/src/com/fallenflame/game/GameEngine.java:512]: See my PR comment. There was a bug in the fail screen where sometimes the "Game Over" text was Cyan as expected, but sometimes it was white or even like faded gray (idk). Noticed it while running through the game/testing on this branch. It didn't seem worth a whole PR for that one fix, so I just threw it in here. It has nothing to do with the Credits scene, but fixes the other issue. I can make a separate PR for it if you want, but for one line... eh?

**Review by BryanGraeser — APPROVED** [review]
Resolved merge conflicts. Should be GTG now

**Pull request merged**